packages/bootstrap/scss/_variables.scss

```diff
@@ -964,6 +964,26 @@ $progressbar-indeterminate-gradient: null !default;
 
 $progressbar-chunk-border: $bg-color !default;
 
+// Scrollview
+
+$scrollview-pager-offset: 20px !default;
+$scrollview-pager-item-spacing: 20px !default;
+$scrollview-pager-item-border-width: 0 !default;
+
+$scrollview-pagebutton-size: 8px !default;
+$scrollview-pagebutton-bg: rgba($carousel-indicator-active-bg, .5) !default;
+$scrollview-pagebutton-border: null !default;
```

Comment:
Why setting border to `null`, instead of `0 !default`? I think we set defaults to null only for `background-color`, `color`, etc. Is this variable used for border-color?

Reply:
As far as I know, when `null` is assigned to a property in the `.scss` files, the same line/rule is not transferred to the `css` file. And this is making our production files smaller.

However, I'm not 100% sure about that :D @joneff could you please confirm or deny this?

Reply:
Here's a confirmation:
![image](https://user-images.githubusercontent.com/90405/48611097-2dedff00-e98e-11e8-96ea-4411f6af7dd1.png)

Note that both values have their use-cases: `null` can reduce output, and `0` can override existing borders.

Reply:
yes, the question here is do we need to override existing border on the pagebutton?

Reply:
We do not need to override existing borders in the Bootstrap theme. Actually, we do not use these variables at all in this theme. However, I added them in order to have consistency with the other `_variables.scss` files. 

Do you think this approach is okay or we should remove the unneeded variables, and if we need them in the future - add them?

Reply:
Imho, setting to `null` is better approach.